auction-server/api-types/src/opportunity.rs

```diff
@@ -343,59 +343,59 @@ pub enum OpportunityParamsV1ProgramSvm {
 #[derive(Serialize, Deserialize, ToSchema, Clone, PartialEq, Debug, ToResponse)]
 #[serde(rename_all = "snake_case")]
 pub enum FeeToken {
-    InputToken,
-    OutputToken,
+    SearcherToken,
+    UserToken,
 }
 
 #[serde_as]
 #[derive(Serialize, Deserialize, ToSchema, Clone, PartialEq, Debug, ToResponse)]
 #[serde(tag = "side_specified")]
 pub enum QuoteTokens {
-    #[serde(rename = "input")]
-    #[schema(title = "input_specified")]
-    InputTokenSpecified {
-        /// The token that the user wants to receive
+    #[serde(rename = "searcher")]
+    #[schema(title = "searcher_specified")]
```

Comment:
curious why the schema attribute and the serde attribute don't use the same name

Reply:
i think it's because the serialization to JSON results in something like

```
{
    "side_specified": "searcher",
    ...
}
```

while the schema looks like

```
{
    "title": "searcher_specified",
}
```

there may be a way to coalesce them. i haven't looked too into this

Reply:
I still feel like this is wrong, everywhere else in the code that we have the 
#[serde(rename="...")]
#[schema(title = "..."]
we pass the same thing to both.
I think this will make the /docs endpoint inconsisten with the actual json expected.

Reply:
ok I finally get it!!